cfme/tests/control/test_actions.py

```diff
@@ -596,23 +595,20 @@ def finalize():
     )
 
 
+@pytest.mark.meta(blockers=[BZ(1748410, forced_streams=["5.10"])], automates=[1748410])
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], scope="module")
-@pytest.mark.meta(
-    blockers=[
-        BZ(
-            1549529,
-            forced_streams=["5.10", "upstream"],
-            unblock=lambda provider: provider.one_of(VMwareProvider),
```

Comment:
Don't you want to unblock the test for Vmware ? The bug is only for RHV right ?

Reply:
Oh good catch, thanks @sshveta!